vos/perftest/global/testers.py

```diff
@@ -151,4 +151,4 @@ def _get_next_priv_uri(self):
     gt = GlobalTester()
     # gt.anon_get_pub()
     # gt.anon_get_priv()
-    gt.anon_put()
+    #gt.anon_put()
```

Comment:
This test will never succeed unless it's a test for a failure to `PUT`.

Reply:
It's a test for a failure. The entire `main` is just for debugging. e.g. stepping through.